File: app/src/main/java/com/goldmines/JsonPojos/Answers.java

package com.goldmines.JsonPojos;

import java.util.List;
import javax.annotation.Generated;
import com.google.gson.annotations.SerializedName;

@Generated("net.hexar.json2pojo")
@SuppressWarnings("unused")
public class Answers {

    @SerializedName("AnswerItem")
    private List<AnswerItem> mAnswerItem;

    public List<AnswerItem> getAnswerItem() {
        return mAnswerItem;
    }

    public void setAnswerItem(List<AnswerItem> answerItem) {
        mAnswerItem = answerItem;
    }

}

File: app/src/main/java/com/goldmines/jsonkotlinpojos/Answers.kt
package com.goldmines.jsonkotlinpojos


import com.google.gson.JsonDeserializer
import com.google.gson.annotations.SerializedName
import java.util.*

data class Answers(
        @SerializedName("AnswerItem")
        val answerItem: List<AnswerItem>)
{

        internal var deserializer: JsonDeserializer<AnswerItem> = JsonDeserializer { json, typeOfT, context ->
                val jsonObject = json.asJsonPrimitive

                if()



        }

}